src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -352,6 +349,31 @@ public override void Execute()
                 this.VirtualNetworkGateway.VpnClientConfiguration.AadAudience = null;
             }
 
+            if (this.VirtualNetworkGateway.VpnClientConfiguration?.VpnAuthenticationTypes != null && this.VirtualNetworkGateway.VpnClientConfiguration.VpnAuthenticationTypes.Count() > 0)
```

Comment:
just wondering, why we dnt have this login in New command let?

Reply:
we let the nrp handle it for new cmd. but for update we have to reset the previously set values.eg: auth: aad updated to auth: radius. this will throw error in update saying unused param aad is se